webpack.config.js

```diff
@@ -52,7 +52,12 @@ module.exports = {
         query: {
           presets: ['react', 'es2015']
         }
-      }
+      },
+      { test: /\.css$/,  loader: "style-loader!css-loader" },
+      { test: /\.less$/, loader: "style-loader!css-loader!less-loader" },
+      { test: /\.gif$/, loader: "url-loader?mimetype=image/png" },
+      { test: /\.woff(2)?(\?v=[0-9].[0-9].[0-9])?$/, loader: "url-loader?mimetype=application/font-woff" },
```

Comment:
So what is going on with the font replacements here?
We can also include them in the stylesheet if you want, that way we separate the concerns.